src/main/java/matcha/exception/MatchaException.java

```diff
@@ -0,0 +1,24 @@
+package matcha.exception;
+
+/**
+ * Represents a custom exception for all errors related to the Matcha program
+ */
+public class MatchaException extends Exception {
+    /**
+     * Constructs a new MatchaException with the error message
+     * @param msg the error message
+     */
+    public MatchaException(String msg) {
```

Comment:
Good attempt in incorporating the name of the chatbot to classes' names. However, it may be confusing to some who may not be sure that Meowception is an Exception.